### ISSUE TITLE
[Quest][Zul'Drak] Cocooned!

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4647]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Some cocoons have 2-4 trolls in them when it should only be 1, some may be empty.
**Expected behaviour**: Tell us what should happen instead
Break a cocoon get a troll or a crusader.
**Steps to reproduce the problem**:

1. Open cacoon see what you get.
2. 
3. 

**Include proofs for this behaviour**
[Video Link](https://www.youtube.com/watch?v=QF-xBxH4iVw)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Quest](http://www.wowhead.com/quest=12606/cocooned)
[Cocoon](http://www.wowhead.com/npc=28413/nerubian-cocoon)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_032917_164829](https://cloud.githubusercontent.com/assets/12252441/24475951/a17dc808-149f-11e7-9eb3-303b33f199c5.jpg)
![wowscrnshot_032917_164730](https://cloud.githubusercontent.com/assets/12252441/24475949/a1527cc0-149f-11e7-9440-1bfe4304036e.jpg)

**Comment by Rushor** [comment]
realted: https://github.com/TrinityCore/TrinityCore/issues/16357

**Comment by Rushor** [comment]
going to add a corescript for this for manual summon

**Comment by Rushor** [comment]
@TheRealestMcCoy could you please link to http://wotlk.openwow.com with quests/npcs and so on (except if you want to provide scrrenshots/comments/proofs) thanks :)

**Comment by Rushor** [comment]
Scripts/Northrend: Quest Cocooned! - Add civilian flag, remove experience and spawn only one Captive/Footman on death

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4647
* commit: e5d0f99fa7df8f3afa86d1b75cb999685406c3b2

![GitHub Logo](http://i.imgur.com/e9XGmst.jpg)
![GitHub Logo](http://i.imgur.com/B8APNLU.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
